client/src/components/relatedItems/Modal.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+
+
+export default class Modal extends React.Component {
+  constructor(props) {
+    super(props);
+    // this.el = document.createElement('div');
+    this.state = {
+      modal: document.createElement('div')
+    }
```

Comment:
I'm wondering if you need to keep the element itself inside this.state. It doesn't seem to update or change ever, so maybe using a property (`this.modal = document.createElement('div')`) would make more sense?

Reply:
Yes, it worked. I thought if something changed in componentDidMount, it should relate to state change. `this.setState` works on it